### PR TITLE
Update expiration doc

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Docu update: expired Kyma clusters allow the update ~~of the "context" attribute~~. Only new **globalaccount_id** is required.
- Use ordered lists for procedures.
- Fix a link.
- Fix a prerequisites list.

See #935

### CODE REVIEW

File: docs/contributor/03-30-trial-and-free-expiration.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Details
+
+The cleanup CronJob triggers the trial instance expiration by sending a `PUT` request to `/expire/service_instance/{instanceID}` KEB API endpoint, where `instanceID` must be a trial or free instance ID. The possible KEB responses are:
+
+| Status Code | Description                                                                                             |
+| --- |---------------------------------------------------------------------------------------------------------|
+| 202 Accepted | Returned if the Service Instance expiration has been accepted and is in progress.                       |
+| 400 Bad Request | Returned if the request is malformed, missing mandatory data, or when the instance's plan is not Trial or Free. |
+| 404 Not Found | Returned if the instance does not exist in database.                                                    |
```
Comment (mmitoraj) [inline docs/contributor/03-30-trial-and-free-expiration.md:19]:
```suggestion
| 404 Not Found | Returned if the instance does not exist in the database.                                                    |
```

File: docs/contributor/03-30-trial-and-free-expiration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+If KEB accepts the instance expiration request, then it marks the instance as expired by populating the instance's `ExpiredAt` field with a timestamp when the request is accepted. Then, it creates a suspension operation. After the suspension operation is added to the operations queue, KEB sets the **parameters.ers_context.active** field to `false`. The instance is deactivated and no longer usable. It can only be removed by deprovisioning request.
```
Comment (mmitoraj) [inline docs/contributor/03-30-trial-and-free-expiration.md:21]: Have you considered using an ordered list to describe the flow? It has many "thens" and "afters". Please ignore my comment if you don't find it useful.

File: testing/e2e/provisioning/README.md [hunk 3]
```diff
@@ -8,11 +8,9 @@ The test is executed on a dev cluster. It is executed after every merge to the `
 
 ## Prerequisites
 
-To run this test, you must have the following Secrets inside your cluster:
 - Gardener Secret per provider
 - Service Manager Secret
-
-You must also have Kyma Environment Broker [configured](https://github.com/kyma-project/kyma-environment-broker/tree/main#configuration) to use these Secrets to create Kyma runtime successfully.
+- Kyma Environment Broker [configured](https://github.com/kyma-project/kyma-environment-broker/tree/main#configuration) to use these Secrets to create Kyma runtime successfully
```
Comment (mmitoraj) [inline testing/e2e/provisioning/README.md:13]: There is no such anchor in the main README.md. Is the link correct?

File: docs/contributor/03-30-trial-and-free-expiration.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Trial and Free Instance Expiration
+
+## Overview
+
+You can explore and use SAP BTP, Kyma runtime for free for a limited period with the following plans:
+* The trial service plan for 14 days.
+* The free plan for 30 days.
+
```
[the remark this answers is not in the document]
Comment (IwonaLanger) [reply docs/contributor/03-30-trial-and-free-expiration.md:8]: As discussed, I'll keep the current version here.

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**